keps/sig-scheduling/4816-dra-prioritized-list/README.md

```diff
@@ -0,0 +1,1128 @@
+-->
+
+Start of v1.32 development cycle (v1.32.0-alpha.1-178-gd9c46d8ecb1):
+
+- `k8s.io/dynamic-resource-allocation/cel`: 88.8%
+- `k8s.io/dynamic-resource-allocation/structured`: 82.7%
+- `k8s.io/kubernetes/pkg/controller/resourceclaim`: 70.0%
+- `k8s.io/kubernetes/pkg/scheduler/framework/plugins/dynamicresources`: 72.9%
+
+##### Integration tests
+
+<!--
+Integration tests are contained in k8s.io/kubernetes/test/integration.
+Integration tests allow control of the configuration parameters used to start the binaries under test.
+This is different from e2e tests which do not allow configuration of parameters.
+Doing this allows testing non-default options and multiple different and potentially conflicting command line options.
+-->
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+-->
+
+The existing [integration tests for kube-scheduler which measure
+performance](https://github.com/kubernetes/kubernetes/tree/master/test/integration/scheduler_perf#readme)
+will be extended to cover the overheaad of running the additional logic to
+support the features in this KEP. These also serve as [correctness
+tests](https://github.com/kubernetes/kubernetes/commit/cecebe8ea2feee856bc7a62f4c16711ee8a5f5d9)
+as part of the normal Kubernetes "integration" jobs which cover [the dynamic
+resource
+controller](https://github.com/kubernetes/kubernetes/blob/294bde0079a0d56099cf8b8cf558e3ae7230de12/test/integration/scheduler_perf/util.go#L135-L139).
+
+##### e2e tests
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+
+We expect no non-infra related flakes in the last month as a GA graduation criteria.
+-->
+
+End-to-end testing depends on a working resource driver and a container runtime
+with CDI support. A [test
+driver](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/dra/test-driver)
+was developed as part of the overall DRA development effort. We will extend this
+test driver to enable support for alternative device requests and add tests to
+ensure they are handled by the scheduler as described in this KEP.
+
+### Graduation Criteria
+
+#### Alpha
+
+- Feature implemented behind a feature flag
+- Implemented in the scheduler but not necessarily the cluster auto scaler
+- Initial e2e tests completed and enabled
+
+#### Beta
+
+- Gather feedback
+- Implement node scoring
+- Cluster auto scaler implementation
```

Comment:
In light of the discussion in the other thread, I think this should be "Evaluate the feasibility of an implementation in cluster-autoscaler". Otherwise I would see it unlikely to graduate this to beta in v1.33 as the current proposal intends.

Reply:
done